daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ModParser.scala

```diff
@@ -141,6 +141,7 @@ private[parser] class ModParser[P](parameters: ParserParameters[P]) {
           name,
           !choiceTags(nonConsumingTag),
           controllers,
+          observers = None, //ENil(TBuiltin(BTParty)), //NICK, extend parser for (optional?) observers.
```

Comment:
If you do not plan to fix the parser in this PR, please add a FIXME with a github issue.